<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoMovieWindow</class>
 <widget class="QDialog" name="infoMovieWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>705</width>
    <height>436</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>705</width>
    <height>436</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="favouriteButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Favourite</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="willWatchButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Will watch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="watchedButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Watched</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="text">
    <string>film description</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="languageLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Language:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="languageLabel_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>211</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="genreLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Genre:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="genreLabel_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="ratingLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Rating:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ratingLabel_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QLabel" name="durationLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Duration:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="durationLabel_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="yearLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Year:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="yearLabel_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <underline>false</underline>
       </font>
      </property>
      <property name="text">
       <string>value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="userRaitingLabel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>323</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
